add missing Issues content for Review List GA (#3255)
getsentry/sentry-docs: src/docs/product/error-monitoring/issue-details/index.mdx

---
title: Issue Details
sidebar_order: 10
description: "Learn more about what information displays on the Issue Details page to help you efficiently triage an issue."
---

> The Issue Details page helps you to gain further insight into the source of the issue. The information that is provided on the primary Issues page also displays on the Issue Details page. This page displays information about a specific error event that's part of an issue, however, you can navigate between the events of an issue using the "Older" and "Newer" buttons.

Reviewer: Suggested change here:

> This page helps you understand the source of an issue and the impact it has on your app’s users. The left-hand side contains information about the latest event such as the stacktrace and which specific user and browser was detected. The right-hand side is a summary of all the events grouped together by this issue.

> 

Reviewer: What does `The information that is provided on the primary Issues page also displays on the Issue Details page` mean? I think we should cut that.

TO DO Add screenshot here of Issue Details page

Click on an issue on the main Issues page to open the Issue Details page.

> This page displays the Details tab when it first opens, but several other tabs are available:

Reviewer: I don’t think we should describe the tabs first, but jump into what info is on the first page.

- Activity: A history of activity for the event where team members can share comments.
- User Feedback: Any feedback collected from users through the <PlatformLink to="/enriching-events/user-feedback/">SDK's user feedback</PlatformLink>.
- <PlatformLink to="/enriching-events/attachments/">Attachments</PlatformLink>: Stored additional files, such as config or log files that are attached to the event.
- Tags: More detailed information about the tags across all the events included in the issue.
- Events: A list of all the events included in the issue.
- Merged Issues: A list of issues that have been merged with this one.
- Similar Issues: A list of issues that have a similar stack trace.

> In the right hand sidebar, [sentry.io](https://sentry.io) reflects how often the issue has occurred in the last 24 hours and the last 30 days, as well as the last time and the first time the issue was seen. If the issue is linked to any GitHub or Jira issues, that's displayed here as well. You can also use this section to link to existing GitHub or Jira issues. Lastly, there is a facet map, which shows the distribution of values for the tags for all events included in the issue. All of these values are based on the environment that you've selected in the Global Header.

Reviewer: We should explain the stacktrace first, this can come later.

In addition, this page provides several other key pieces of information.

## Suspect Commits

> If you've configured [suspect commits](/product/releases/suspect-commits/), these are displayed. This allows you to immediately investigate commits that are likely to have generated the issue. The owners of suspect commits can be suggested assignees for an issue as well.

Reviewer: We should explain suspect commits first, then explain how to configure them.

## Tags

> Tags are the diagnostic information set by the SDK for the individual event. These are specific to each event in an issue. You can set your own tags to make them more useful for debugging as described in <PlatformLink to="/enriching-events/tags/">Customize Tags</PlatformLink>.

Reviewer: We should explain how this relates to the tags section in the sidebar.

Author: Edited

> Tags are the diagnostic information set by the SDK for the individual event. These are specific to each event in an issue. You can set your own tags to make them more useful for debugging as described in <PlatformLink to="/enriching-events/tags/">Customize Tags</PlatformLink>.

Author: ```suggestion
[Tags](/product/sentry-basics/search/) are key/value string pairs that are both indexed and searchable. For example, a tag provides you with information such as the browser, device, or user associated with the event. Tags are the diagnostic information sent by the SDK for the individual event. 

The tags displayed in the main section of this page are specific to the event that you're viewing. In contrast, the tags displayed in the right-hand sidebar are a summary of all tag values for all events included in the issue. You can set your own tags to make them more useful for debugging as described in <PlatformLink to="/enriching-events/tags/">Customize Tags</PlatformLink>.
```

> ## Stack Trace

Author: ```suggestion
## Exception (Stack Trace)
```

> 

Reviewer: Image required here

> The Exception section of the page displays the stack trace, which shows you the line of code that all of the events in the issue errored on. It's the most important piece of information that the Sentry grouping algorithm uses to group events into one issue.

Author: ```suggestion
The Exception section of the page displays the stack trace, which shows you the line of code that all of the events in the issue errored on. 

TO DO add image here

It's the most important piece of information that the Sentry grouping algorithm uses to group events into one issue.
```

## Breadcrumbs

> Breadcrumbs provide a history and timeline leading up to the error event. These can include things like HTTP requests, log statements from the console or server, and DOM (document object model) events in JavaScript. You can set your own <PlatformLink to="/enriching-events/breadcrumbs/">breadcrumbs</PlatformLink> to make them more useful for debugging.

Reviewer: Image required here

> Breadcrumbs provide a history and timeline leading up to the error event. These can include things like HTTP requests, log statements from the console or server, and DOM (document object model) events in JavaScript. You can set your own <PlatformLink to="/enriching-events/breadcrumbs/">breadcrumbs</PlatformLink> to make them more useful for debugging.

Author: ```suggestion
Breadcrumbs provide a history and timeline leading up to the error event. These can include things like HTTP requests, log statements from the console or server, and DOM (document object model) events in JavaScript. 

TO DO add image here

You can set your own <PlatformLink to="/enriching-events/breadcrumbs/">breadcrumbs</PlatformLink> to make them more useful for debugging.
```

> 

Reviewer: Need a section that describes/links to Linked Issues from here and why they’re useful

Author: Added link in p about right hand sidebar

## Trace Details

> The Headers section of the page displays metadata about the host that produced the issue, including user, browser, operating system, SDK and version, and trace details. [Tracing](/product/performance/distributed-tracing/) augments your existing error data by capturing interactions between your software systems. You can click the "Search by Trace" button to see all the events that share the same trace ID.

Reviewer: Image required here

> The Headers section of the page displays metadata about the host that produced the issue, including user, browser, operating system, SDK and version, and trace details. [Tracing](/product/performance/distributed-tracing/) augments your existing error data by capturing interactions between your software systems. You can click the "Search by Trace" button to see all the events that share the same trace ID.

Author: ```suggestion
[Tracing](/product/performance/distributed-tracing/) augments your existing error data by capturing interactions between your software systems. In the Trace Details section, you can click the "Search by Trace" button to see all the events that share the same trace ID.
```

---
title: Issue Details
sidebar_order:

> description: "Learn more about Issue Details page."

Reviewer 2: ```suggestion
description: "Learn more about what information displays on the Issue Details page to help you efficiently triage an issue."
```

---

> The Issue Details page provides you with information to help you to get further insight into the reasons for the issue. All the informtion about the issue that's on the Issues page is displayed on the Issue Details page as well. This page displays information about a specific error event that's part of an issue, however, you can navigate between the events of an issue using the "Older" and "Newer" buttons.

Reviewer 2: ```suggestion
The Issue Details page helps you to gain further insight into the source of the issue. The information that is provided on the primary Issues page also displays on the Issue Details page. This page displays information about a specific error event that's part of an issue, however, you can navigate between the events of an issue using the "Older" and "Newer" buttons.
```

TO DO Add screenshot here of Issue Details page

> To access this page, click the any issue on the Issues page.

Reviewer 2: ```suggestion
Click on an issue on the main Issues page to open the Issue Details page.
```

> This page opens on the Details tab, but there are several other tabs as well:

Reviewer 2: ```suggestion
This page displays the Details tab when it first opens, but several other tabs are available:
```

> - Activity: A history of activity on the for the event where team members can post comments.

Reviewer 2: ```suggestion
- Activity: A history of activity for the event where team members can share comments.
```

> - User Feedback: Any feedback collected from users through Sentry's User Feedback widget. TO DO Add link here to take them to Platform selection page but that gets them to User Feedback

Reviewer 2: FYI - we have both the User Feedback widget and the API, so suggest rephrasing as edited here. Also added the platform redirect link (which uses the context of the reader's platform, if known, to direct them to the correct content. If not known, the platform selector intervenes, they select the platform and are then directed to the User Feedback content):
```suggestion
- User Feedback: Any feedback collected from users through the <PlatformLink to="/enriching-events/user-feedback/">SDK's user feedback</PlatformLink>. 
```

> - Attachments: Stored additional files, such as config or log files that are attached to the event. TO DO Add link here to take them to Platform selection page but that gets them to Attachments

Reviewer 2: ```suggestion
- <PlatformLink to="/enriching-events/attachments/">Attachments</PlatformLink>: Stored additional files, such as config or log files that are attached to the event. 
```

- Tags: More detailed information about the tags across all the events included in the issue.
- Events: A list of all the events included in the issue.
- Merged Issues: A list of issues that have been merged with this one.
- Similar Issues: A list of issues that have a similar stack trace.

In the right hand sidebar, [sentry.io](https://sentry.io) reflects how often the issue has occurred in the last 24 hours and the last 30 days, as well as the last time and the first time the issue was seen. If the issue is linked to any GitHub or Jira issues, that's displayed here as well. You can also use this section to link to existing GitHub or Jira issues. Lastly, there is a facet map, which shows the distribution of values for the tags for all events included in the issue. All of these values are based on the environment that you've selected in the Global Header.

> There are several other key pieces of information on the page.

Reviewer 2: There is/there are constructions tend to lead to dead writing
```suggestion
In addition, this page provides several other key pieces of information.
```

## Suspect Commits

> If you've configured [suspect commits](product/releases/suspect-commits/), these are displayed. This allows you to immediately investgate commits that are likely to have generated the issue. The owners of suspect commits can be suggested assignees for an issue as well.

Reviewer 2: ```suggestion
If you've configured [suspect commits](/product/releases/suspect-commits/), these are displayed. This allows you to immediately investigate commits that are likely to have generated the issue. The owners of suspect commits can be suggested assignees for an issue as well.
```

## Tags

> Tags are the diagnostic information set by the SDK for the individual event. These are specific to each event in an issue. You can set your own tags to make them more useful for debugging.
> 
> TO DO Add link here to take them to Platform selection page but that gets them to info about custom tags (Enriching Events > Context??)

Reviewer 2: ```suggestion
Tags are the diagnostic information set by the SDK for the individual event. These are specific to each event in an issue. You can set your own tags to make them more useful for debugging as described in <PlatformLink to="/enriching-events/tags/">Customize Tags</PlatformLink>.
```

## Stack Trace

> In the Exception section of the page displays the stack trace, which shows you the line of code that all of the events in the issue errored on. It's the most important piece of information that the Sentry grouping algorithm uses to group events into one issue.

Reviewer 2: ```suggestion
The Exception section of the page displays the stack trace, which shows you the line of code that all of the events in the issue errored on. It's the most important piece of information that the Sentry grouping algorithm uses to group events into one issue.
```

## Breadcrumbs

> The breadcrumbs are the history and timeline leading up to error event. These can include things like HTTP requests, log statements from the console or server, and DOM (document object model) events in JavaScript. You can set your own breadcrumbs to make them more useful for debugging.

Reviewer 2: ```suggestion
Breadcrumbs provide a history and timeline leading up to error event. These can include things like HTTP requests, log statements from the console or server, and DOM (document object model) events in JavaScript. You can set your own <PlatformLink to="/enriching-events/breadcrumbs/">breadcrumbs</PlatformLink> to make them more useful for debugging.
```

> TO DO Add link here to take them to Platform selection page but that gets them to Breadcrumbs

Reviewer 2: ```suggestion

```

---
title: Issue Details
sidebar_order: 10
description: "Learn more about what information displays on the Issue Details page to help you efficiently triage an issue."
---

> The Issue Details page helps you to gain further insight into the source of the issue and the impact it has on your application's users. The main area of the page displays information about a specific error event that's part of an issue. The right-hand sidebar of the page displays a summary of all the events grouped together in this issue.

Reviewer 3: right-hand sidebar AND the top panel

TO DO Add screenshot here of Issue Details page

> Click on an issue on the main Issues page to open the Issue Details page.

Reviewer 3: probably unnecessary?

Author: I think the action of clicking is pretty intuitive, but this sentence is more about ensuring that the user understands that the page they get to when they click on an issue is called the Issue Details page, since that name isn't anywhere on the actual UI, but we use this naming convention for it.
